src/messaging/ExchangeMgr.h

```diff
@@ -229,6 +232,7 @@ class DLL_EXPORT ExchangeManager : public SessionManagerDelegate
     uint16_t mNextExchangeId;
     uint16_t mNextKeyId;
     State mState;
+    bool mIsActiveStateForced = false; // Active state forced despite there is currently no active exchange
```

Comment:
Right, so this is trying to work around the bug I mention.... but then we can still have a closing commissioning window put us back into idle mode even if we have an active exchange.

Reply:
Removed after introducing the fast polling consumers counter